Makefile.in

```diff
@@ -21,7 +21,8 @@ endif
 HAS_NDPI=$(shell pkg-config --exists libndpi; echo $$?)
 ifeq ($(HAS_NDPI), 0)
     NDPI_INC = $(shell pkg-config --cflags libndpi | sed -e 's/\(-I[^ \t]*\)/\1\/libndpi/g')
-    NDPI_LIB = $(shell pkg-config --libs libndpi)
+    # Use static libndpi library as building against the dynamic library fails
+    NDPI_LIB = -Wl,-Bstatic $(shell pkg-config --libs libndpi) -Wl,-Bdynamic
```

Comment:
`-Wl,-Bstatic` is duplicated

Reply:
The default behavior is to use the dynamic libraries. With "-Wl,-Bstatic" it switches to static for the following libraries. With the "-Wl,-Bdynamic" it switches back to its default behavior making libndpi the only exception.